spec/spec_helper.rb

```diff
@@ -1,31 +1,70 @@
+ENV["RAILS_ENV"] ||= 'test'
+ENV["AUTOMATE_DOMAINS"] = "ManageIQ" # Reset only the ManageIQ automate domain when testing.
+
+require File.expand_path("../manageiq/config/environment", __FILE__)
+require 'rspec/rails'
 require 'simplecov'
-SimpleCov.start
 
-require 'manageiq-content'
+SimpleCov.start
 
-# Reset only the ManageIQ automate domain when testing.
-ENV["AUTOMATE_DOMAINS"] = "ManageIQ"
+if ENV["TRAVIS"] || ENV['CI']
+  require 'coveralls'
+  Coveralls.wear!('rails') { add_filter("/spec/") }
+end
 
 def require_domain_file
-  spec_name = caller_locations.first.path
+  spec_name = caller_locations(1..1).first.path
   file_name = spec_name.sub("spec/", "").sub("_spec.rb", ".rb")
-
   AutomateClassDefinitionHook.require_with_hook(file_name)
 end
 
 def domain_file
   caller_locations(1..1).first.path.sub("spec/", "").sub("_spec.rb", ".rb")
 end
 
+# Requires supporting ruby files
+Dir[Rails.root.join('spec', 'support', '**', '*.rb')].each { |f| require f }
 Dir[ManageIQ::AutomationEngine::Engine.root.join("spec/support/**/*.rb")].each { |f| require f }
 Dir[ManageIQ::Content::Engine.root.join("spec/support/**/*.rb")].each { |f| require f }
 
 RSpec.configure do |config|
   config.include Spec::Support::AutomationHelper
+  config.include Spec::Support::RakeTaskExampleGroup, :type => :rake_task
+
+  config.fixture_path = "#{::Rails.root}/spec/fixtures"
+  config.use_transactional_fixtures = true
+  config.use_instantiated_fixtures  = false
+
+  # From rspec-rails, infer what helpers to mix in, such as `get` and
+  # `post` methods in spec/controllers, without specifying type
+  config.infer_spec_type_from_file_location!
+
+  config.expect_with :rspec do |c|
+    c.syntax = :expect
+  end
+
+  config.mock_with :rspec do |c|
+    c.allow_message_expectations_on_nil = false
+    c.syntax = :expect
+  end
 
-  config.around(:example) do |ex|
+  config.before(:suite) do
+    if ENV["TRAVIS"] && ENV["TEST_SUITE"] == "vmdb"
+      require Rails.root.join('spec', 'coverage_helper.rb')
```

Comment:
I don't think we want to run this here.  It helps verify test coverage on the core rails app.

Reply:
Also, `ENV["TEST_SUITE"]` is never set in this repo.